### PR TITLE
Refactor dApp Structure 04 | Removes/Updates Page Components

### PR DESCRIPTION
**Pull request by Da-Colon**
## Description

There is probably some left over hooks but this takes care of any other screens that needed to be updated to use the new FractalProvider or be removed.

### CODE REVIEW

File: src/pages/Treasury/index.tsx [hunk 1]
```diff
@@ -21,13 +20,15 @@ function TableRowWrapper({ children }: { children?: ReactNode }) {
...
+  const {
+    gnosis: { safe },
+    treasury: { assetsFungible, assetsNonFungible },
+  } = useFractal();
 
   const { t } = useTranslation('treasury');
   return (
     <div>
-      <H1>{t('titleTreasury', { daoName: state.name })}</H1>
+      <H1>{t('titleTreasury', { daoName: '' })}</H1>
```
Comment (herbig) [inline src/pages/Treasury/index.tsx:31]: @Da-Colon I realize we don't have name actually loading yet, but does the `gnosis` `safe` have a name field to reference here?

File: src/pages/Treasury/index.tsx [hunk 2]
```diff
@@ -180,9 +181,9 @@ function Treasury() {
           ))}
         </div>
       </div>
-      {!gnosisAssetsFungible!.length && !gnosisAssetsNonFungible.length && (
+      {!assetsFungible!.length && !assetsNonFungible.length && (
         <div className="px-1">
-          <ContentBanner description={t('descTreasury', { daoName: state.name })} />
+          <ContentBanner description={t('descTreasury', { daoName: '' })} />
```
Comment (herbig) [inline src/pages/Treasury/index.tsx:186]: same with this, it seems like we should have a name field in state somewhere even if it's not there yet

**Review by herbig — COMMENTED** [review]
just questions about the name field! Looks good otherwise

File: src/pages/Treasury/index.tsx (hunk 1 above)
Comment (Da-Colon) [reply src/pages/Treasury/index.tsx:31]: not at the moment nope.

Comment (herbig) [reply src/pages/Treasury/index.tsx:31]: @mudrila could you make note to put the name in here once your work on the name contract is good to go

**Review by tbwebb22 — APPROVED** [review]
LGTM

**Pull request merged**